### PR TITLE
1.5 release notes

### PR DESCRIPTION
**Pull request by mairin**
### What does this PR do?

Creates blog post for 🦭 Podman Desktop's 1.5 release 

### How to test this PR?

In the rootdir of a checkout of this pr type `yarn website:prod`

### CODE REVIEW

File: website/blog/2023-10-26-release-1.5.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+When creating a container from the Images list, there's now an option to provide an environment file to set env variables for the new container. [#4026](https://github.com/containers/podman-desktop/pull/4026) &amp;&amp; [#4025](https://github.com/containers/podman-desktop/pull/4025)
+
+### Enhancements to the Settings area
+
+The user experience for enabling or disabling Docker compatibility is improved, with a new entry in the Settings > Preferences screen that includes contextual guidance. [#4093](https://github.com/containers/podman-desktop/pull/4093)
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/270497318-902b2566-62ad-4ee6-87af-6a9a2705de99.mov" width='100%' height='100%' />
+
+Enhanced logic for displaying or hiding properties listed under the Settings > Preferences screens is now available. [#4159](https://github.com/containers/podman-desktop/pull/4159)
```
Comment (deboer-tim) [inline website/blog/2023-10-26-release-1.5.md]: Needs a 'why' (e.g. how we're using it), or maybe just move to the extensions API section.

File: website/blog/2023-10-26-release-1.5.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+---
+
+## Other Notable Enhancements
+
+<!-- CATCHALL SECTION FOR MINOR ENHANCEMENTS -- >
+<!-- USE BULLET POINTS -->
+<!-- EXAMPLE -->
+<!-- - Show warning when creating a pod with two containers that use the same port [#2671](https://github.com/containers/podman-desktop/pull/2671). -->
+
+- The progress of loading an image into kind is now visible as a task in the Podman Desktop task manager. [#4061](https://github.com/containers/podman-desktop/pull/4061)
```
Comment (deboer-tim) [inline website/blog/2023-10-26-release-1.5.md]: Capitalize Kind. "as a task in the Podman Desktop task manager" is a bit redundant, maybe just "in the task manager".

File: website/blog/2023-10-26-release-1.5.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing a new onboarding feature that we hope will earn your seal of approval! But wait... there's so much more!
```
Comment (deboer-tim) [inline website/blog/2023-10-26-release-1.5.md]: How did none of us use this before?? Awesome intro.

File: website/blog/2023-10-26-release-1.5.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+- **Onboarding**: Easy setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.1**: [Podman 4.7.1](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: ...
+- **Environment file support**: ...
+- **Enhancements to the Settings area**: ...
+- **Improved user experience for state changes**: ...
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
```
Comment (deboer-tim) [inline website/blog/2023-10-26-release-1.5.md]: boatload: I see what you did here, and +1 for all future release notes including nautical-based puns.

File: website/blog/2023-10-26-release-1.5.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+### Onboarding
+
+**Mo note: I have to fill this in more**
+
+- Notification registry for notifications displayed on the dashboard [#4333](https://github.com/containers/podman-desktop/pull/4333)
+
+### Command Palette
+
+A new, search-driven command palette is now available to enable quick access to various commands available across Podman Desktop. You can try this new tool out by hitting the F1 key in the Podman Desktop UI. [#4081](https://github.com/containers/podman-desktop/pull/4081) &amp;&amp; [#3979](https://github.com/containers/podman-desktop/pull/3979)
```
Comment (deboer-tim) [inline website/blog/2023-10-26-release-1.5.md]: Could remove "in the Podman Desktop UI", it's kinda obvious.

File: website/blog/2023-10-26-release-1.5.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+- The "podify" icon & button on the Containers list was unusually large in release 1.4. That regression has been corrected in this release. [#4122](https://github.com/containers/podman-desktop/pull/4122)
+
+- An error in the instructions for Windows users on migrating from Docker has been corrected. [#4157](https://github.com/containers/podman-desktop/pull/4157)
+
+---
+
+## Community Thank You
+
+🎉 We’d like to say a big thank you to everyone who helped to make Podman Desktop even better. In this
+sprint we received pull requests from the following people:
```
Comment (slemeur) [inline website/blog/2023-10-26-release-1.5.md]:
```suggestion
release we received pull requests from the following people:
```

File: website/blog/2023-10-26-release-1.5.md (hunk 1 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: Good idea, moved to the extensions API section.

File: website/blog/2023-10-26-release-1.5.md (hunk 2 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: Cool, did both :)

File: website/blog/2023-10-26-release-1.5.md (hunk 3 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: I actually went back thru old posts to make sure we hadn't :seal: 🤣

File: website/blog/2023-10-26-release-1.5.md (hunk 4 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: ⛵

File: website/blog/2023-10-26-release-1.5.md (hunk 5 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: done!

File: website/blog/2023-10-26-release-1.5.md (hunk 6 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: good call! ill submit as a fix in the template too.

Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: https://github.com/containers/podman-desktop/pull/4552 👍🏻

File: website/blog/2023-10-26-release-1.5.md [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing **a new onboarding feature** that we hope will earn your seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Guided setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.1**: [Podman 4.7.1](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
```
Comment (benoitf) [inline website/blog/2023-10-26-release-1.5.md]: 4.7.2 now

File: website/blog/2023-10-26-release-1.5.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+<!-- ### Podman v4.5.1 -->
+<!-- Podman Desktop 1.1 moves up to [Podman 4.5.1](https://github.com/containers/podman/releases/tag/v4.5.1). -->
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+**Mo note: I have to fill this in more**
```
Comment (benoitf) [inline website/blog/2023-10-26-release-1.5.md]: Todo ?

File: website/blog/2023-11-03-release-1.5.md [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: Go deep with extended details on Kuberentes pods in the pod "Summary" tab
+- **Environment file support**: Chart out environment variables for new containers to access on creation
+- **Enhancements to the Settings area**: Get your bearings with improved Docker compatibility mode controls
+- **Improved user experience for state changes**: No more dead reckoning on container state with improved visual indication of status
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
+
+Podman Desktop 1.5 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-1-5-hero](img/podman-desktop-release-1.5/onboarding-selkies.png) -->
```
Comment (deboer-tim) [inline website/blog/2023-11-03-release-1.5.md]: Extraneous `-->`, shows up on final page.

Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md]: fixed, thank you!

File: website/blog/2023-11-03-release-1.5.md [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+We are introducing a new feature providing guided flows for the initial setup of specific 🦭 Podman Desktop extensions. Release 1.5 features two new onboarding flows: Podman, and Compose. Visit **<icon icon="fa-solid fa-cog" size="lg"/>Settings > Resources** in order to start these onboarding flows.
+
+![compose-onboarding-start](https://user-images.githubusercontent.com/799683/280276847-ca0558ab-70ad-48cc-8dd5-67e3eb465a62.png)
```
Comment (jeffmaury) [inline website/blog/2023-11-03-release-1.5.md]: Is there a reason we don't show the Podman onboarding. I'm afraid users may be somehow lost as this is documented between lines above

Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md]: laziness. :) I will boot up the mac and grab some. this is a good point.

File: website/blog/2023-11-03-release-1.5.md [hunk 11]
```diff
@@ -0,0 +1,229 @@
...
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing **a new onboarding feature** that we hope will earn your 🦭 seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Guided setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.2**: [Podman 4.7.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: Go deep with extended details on Kuberentes pods in the pod "Summary" tab
```
Comment (cdrage) [inline website/blog/2023-11-03-release-1.5.md]:
```suggestion
- **Expanded "Summary" tab for Kubernetes pods**: Go deep with extended details on Kubernetes pods in the pod "Summary" tab
```

**Review by cdrage — CHANGES_REQUESTED** [review]
One small spelling error.

I also tried previewing and it looks like one of the videos is corrupted / unable to play?

Screenshot 2023-11-03 at 1 01 30 PM

**Comment by mairin** [comment]
> I also tried previewing and it looks like one of the videos is corrupted / unable to play?

@cdrage can you try again? i can't reproduce on my end - works just fine for me - eg

Screenshot from 2023-11-03 13-12-06

File: website/blog/2023-11-03-release-1.5.md (hunk 11 above)
Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md]: fixed, thank you!! 👍🏻

File: website/blog/2023-11-03-release-1.5.md (hunk 10 above)
Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md]: okay I got some screenshots of podman onboarding now and I reworded the text to be specific about how to access each flow / which button to press / etc.

**Review by cdrage — APPROVED** [review]

File: website/blog/2023-11-03-release-1.5.md [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+<!-- ### Podman v4.5.1 -->
+<!-- Podman Desktop 1.1 moves up to [Podman 4.5.1](https://github.com/containers/podman/releases/tag/v4.5.1). -->
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+We are introducing a new feature providing guided flows for the initial setup of specific 🦭 Podman Desktop extensions. Release 1.5 features two new onboarding flows: Podman, and Compose.
```
Comment (benoitf) [inline website/blog/2023-11-03-release-1.5.md:46]: I'm not sure about the Oxford comma after Podman and before and Compose as we only have two items.

File: website/blog/2023-11-03-release-1.5.md [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+
+- It's now more clear which container/pod providers will autostart when 🦭 Podman Desktop starts. Previously autostart had both a global and a per-provider setting. It has been simplified by removing the global setting. [#3840](https://github.com/containers/podman-desktop/pull/3840)
+
+- The "Working with containers" section of the documentation has been reworked and improved. [#3951](https://github.com/containers/podman-desktop/pull/3951)
+
+---
+
+## Notable Bug Fixes
+
+<!-- CATCHALL SECTION FOR BUG FIXES -- >
```
Comment (benoitf) [inline website/blog/2023-11-03-release-1.5.md:151]: I think we can remove template comments

File: website/blog/2023-11-03-release-1.5.md [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+
+- The image details page always listed the image as being "not used" even when it was. This has been corrected. [#3985](https://github.com/containers/podman-desktop/pull/3985)
+
+- Previously, deleting a specific image tag would cause all tags with the same Image ID to be deleted. This has been fixed so only the selected image tag will be deleted. [#3837](https://github.com/containers/podman-desktop/pull/3837)
+
+- Could not view the pod details for some remote Kubernetes cluster pods due to an encoding error. [#4371](https://github.com/containers/podman-desktop/pull/4371)
+
+- Error logs were being tracked separately from the actions they applied to. This has been addressed for kind cluster creation failure [#4427](https://github.com/containers/podman-desktop/pull/4427) and Compose installation failure [#4407](https://github.com/containers/podman-desktop/pull/4407).
+
+- 🦭 Podman Desktop previously did not appear under the "Development" menu when installed via Flatpak; it appeared under "Utilities." 🦭 Podman Desktop now appears under the "Development" menu. [#3911](https://github.com/containers/podman-desktop/pull/3911)
```
Comment (benoitf) [inline website/blog/2023-11-03-release-1.5.md:170]: Maybe tell that's Linux specific ( flatpak may be known only by Linux users)

**Review by benoitf — APPROVED** [review]

File: website/blog/2023-10-26-release-1.5.md (hunk 8 above)
Comment (mairin) [reply website/blog/2023-10-26-release-1.5.md]: i fixed this since :) thx!

File: website/blog/2023-11-03-release-1.5.md (hunk 12 above)
Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md:46]: 👍🏻 you're right not sure what i was doing there, will fix in another pr!

File: website/blog/2023-11-03-release-1.5.md (hunk 13 above)
Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md:151]: fixing in the new pr :)

File: website/blog/2023-11-03-release-1.5.md (hunk 14 above)
Comment (mairin) [reply website/blog/2023-11-03-release-1.5.md:170]: fixing in the new pr, good point!

**Pull request merged**